src/registers/control.rs

```diff
@@ -251,9 +254,13 @@ mod x86_64 {
 
     impl Cr2 {
         /// Read the current page fault linear address from the CR2 register.
+        ///
+        /// # Errors
+        ///
+        /// This method returns a [`VirtAddrNotValid`] error if the CR2 register contains a non-canonical address. Call [`Cr2::read_raw`] to handle such cases.
```

Comment:
This line is a bit long, perhaps do a line break after 100 chars like rustfmt does?

Reply:
Sure, I wrapped it at 100 (although the default in rustfmt is 80, not sure if that has changed over time, the option is unstable after all).

Reply:
Ah, I wasn't aware that there is a separate limit for wrapping comments. I just meant the default [`max_width`](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#max_width) limit that rustfmt applies to code.